src/EnergyPlus/WindowManager.cc

```diff
@@ -6864,88 +7094,130 @@ namespace WindowManager {
 
         WindowTempsForNominalCond(state, ConstrNum, hgap);
 
+        if (!ANY_INTERIOR_SHADE_BLIND(ShadeFlag)) AbsBeamShadeNorm = 0.0;
+
         // Get center-of-glass conductance and solar heat gain coefficient
         // including inside and outside air films
+        Real64 inputU = state.dataMaterial->Material(state.dataConstruction->Construct(ConstrNum).LayerPoint(1)).SimpleWindowUfactor;
+
+        if (WinterSummerFlag == 1 && inputU > 0) { // only compute adjustment ratio when there is valid user input U
+            Real64 CoeffAdjRatio = 1;
+            Real64 hcinRated = state.dataWindowManager->hcin;
+
+            // Adjustment ratio applies to convective film coefficients when input U value is above the limit of the simple glazing nominal U
+            // Representing the nominal highly conductive frame effects. Solved iteratively.
+            while (inputU - NominalConductance > 0.01) {
```

Comment:
Theoretically, I think here this `while` loop need a max iteration number limit (e.g. 10 or 20 times) in case the stop criterion `inputU - NominalConductance <= 0.01` is difficult or impossible to reach, in order to prevent an infinite loop. If the max iteration is reached, exit the loop and take a best reasonable estimate.

Reply:
And also, could this `while` condition be two-sided? I mean could it be abs(inputU - NorminalConductance) > 0.01, i.e. either `inputU - NominalConductance > 0.01` or `NominalConductance - inputU > 0.01`?

Reply:
@jcyuan2020 Good point. I added a max iteration number of 10. The while loop is actually just solving a quadratic equation and the search space reduces exponentially. I tested a series of `inputU` from 6 to 60 and the maximum # iteration needed is 7. So I guess 10 is good enough.

The original point of this feature is we have a physical upper limit of Nominal U with the default film coefficients and a flat plane assumed as the simple glazing. So no matter how large the users set as the U value, the evaluated nominal U (non-adjusted one at the first iteration) would have an upper limit of ~5.8. However, the users require higher overall U values, assuming there can be highly conductive frames that are not considered in the glazing film coefficients.

When user input is small, there's no guarantee that the NominalConductance would be smaller than the inputU (the diff is very small, though), because the Nominal condition is evaluated at ASHRAE winter condition and the simple glazing film coefficients are evaluated based on the NFRC rated condition. But when `Input - NominalConductance  < 0`, we don't have any justification to adjust that.